src/Collateral.hs

```diff
@@ -115,8 +115,8 @@ mkValidator contractInfo@ContractInfo{..} dat rdm ctx = validate
     doesNotContainAdditionalTokens :: Bool
     doesNotContainAdditionalTokens = length (flattenValue $ U.valueToSc interestscvh ctx) < 4
 
-    validateBorrower :: Bool
-    validateBorrower = traceIfFalse "invalid debt amount sent to interest sc" validateDebtAmnt &&
+    validateReturn :: Bool
+    validateReturn = traceIfFalse "invalid debt amount sent to interest sc" validateDebtAmnt &&
                        traceIfFalse "invalid interest amount sent to interest sc" validateInterestAmnt &&
```

Comment:
After the name change, indentation on the lines below is now not correct, please fix that as well.